src/magic_search.js

```diff
@@ -400,9 +401,10 @@ angular.module('MagicSearch')
                 // showMenu and hideMenu depend on foundation's dropdown. They need
                 // to be modified to work with another dropdown implemenation (i.e. bootstrap)
                 $scope.showMenu = function() {
+                    var facetDrop = $element.find('#facet-drop');
```

Comment:
Ah, you're correct! The bootstrap version uses class instead. I'll leave this one be then